### PR TITLE
File memery

### PR DESCRIPTION
**Pull request by isJuhn**
Fixes possibly the worst game in history, Jissen Pachi-Slot Hisshouhou! Hokuto no Ken F - Seikimatsu Kyuuseishu [BLJM60468]
The game refuses to boot unless the reported game data size is exact down to the kilobyte
[BLJM60468] loadable -> ingame (playable?)
image

Could be worth testing more games that crash/freeze when/after installing data

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellGame.cpp [hunk 1]
```diff
@@ -848,7 +865,15 @@ error_code cellGameGetSizeKB(vm::ptr<s32> size)
 		}
 	}
 
-	*size = ::narrow<u32>(fs::get_dir_size(local_dir) / 1024);
+	idm::select<lv2_fs_object, lv2_file>([&](u32, lv2_file& file)
```
Comment (elad335) [inline rpcs3/Emu/Cell/Modules/cellGame.cpp]: Better fo compiler optimzations
```suggestion
	idm::select<lv2_fs_object, lv2_file>([](u32, lv2_file& file)
```

**Comment by isJuhn** [comment]
pushed new changes that fixes The Walking Dead  Survival Instinct
[BLES01779] loadable -> ingame
rpcs3_2019-01-15_03-48-04

**Comment by isJuhn** [comment]
Added a testcase to support the removing and renaming of lv2_fs_objects if the files are not closed in rename and unlink functions [here](https://github.com/isJuhn/ps3tests/blob/master/sys_fs_unlink/expected.txt)
master returns:
```
Sample started.
cellGameDataCheck()
cellGameCreateGameData()
tmp_contentInfoPath: /dev_hdd0/game/_GDATA_8030011646626
tmp_usrdirPath: /dev_hdd0/game/_GDATA_8030011646626/USRDIR
file_path: /dev_hdd0/game/_GDATA_8030011646626/USRDIR/kot.a
ERROR: sys_fs_stat("/dev_hdd0/game/_GDATA_8030011646626/USRDIR/kot.a") returned 0x8001002b
ERROR: sys_fs_open("/dev_hdd0/game/_GDATA_8030011646626/USRDIR/kot.a") returned 0x8001002b
```
PR returns:
```
Sample started.
cellGameDataCheck()
cellGameCreateGameData()
tmp_contentInfoPath: /dev_hdd0/game/_GDATA_8062434094354
tmp_usrdirPath: /dev_hdd0/game/_GDATA_8062434094354/USRDIR
file_path: /dev_hdd0/game/_GDATA_8062434094354/USRDIR/kot.a
ERROR: sys_fs_stat("/dev_hdd0/game/_GDATA_8062434094354/USRDIR/kot.a") returned 0x80010006
new_path: /dev_hdd0/game/_GDATA_8062434094354/USRDIR/kot.2
ERROR: sys_fs_stat("/dev_hdd0/game/_GDATA_8062434094354/USRDIR/kot.a") returned 0x80010006
Sample finished.
```

**Comment by JohnHolmesII** [comment]
@Megamouse @isJuhn Now, I'm normally a fan of shitposting everywhere, but github might be where a line has to be drawn. Sort this out on Discord ffs.

**Comment by nitrohigito** [comment]
good meme

**Comment by digitaldude555** [comment]
Bad meme...

**Comment by uaqlover** [comment]
why didn't Merged to master build ?

**Comment by digitaldude555** [comment]
I dont know.

**Pull request closed without merging**